packages/ocular/src/api/routes/admin/apps/updateApp.ts

```diff
@@ -0,0 +1,46 @@
+import { IsNotEmpty, IsString, IsOptional, IsEnum } from "class-validator";
+import { validator } from "@ocular/utils";
+import { OAuthService, OrganisationService } from "../../../../services";
+import { AppNameDefinitions } from "@ocular/types";
+import { EventBusService } from "../../../../services";
+const { v4: uuidv4 } = require("uuid");
+
+export default async (req, res) => {
+  const validated = await validator(PostAppsReq, req.body);
+  const eventBus_: EventBusService = req.scope.resolve("eventBusService");
+  const loggedInUser = req.scope.resolve("loggedInUser");
+  const organisationService: OrganisationService = req.scope.resolve(
+    "organisationService"
+  );
+
+  const link_id = uuidv4();
+
+  const data = {
+    link: validated.link,
+    link_id,
+    eventBus_,
```

Comment:
Why pass the eventBus_ as a parameter here? We can resolve it inside organizationService.